conf/test_experimental.config

```diff
@@ -29,7 +29,8 @@ params {
     input = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/rnaseq_expression/SRP254919.samplesheet.csv'
     matrix =  'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/rnaseq_expression/SRP254919.salmon.merged.gene_counts.top1000cov.tsv'
     contrasts =  'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/rnaseq_expression/SRP254919.contrasts.csv'
-    pathway = "propd,propd_grea,propr,cor"
+    tools = "$projectDir/assets/tools_samplesheet.csv"
```

Comment:
tools is now declared inside nextflow.config with default value `assets/tools_samplesheet.csv`, so you don't need it here